src/node_contextify.cc

```diff
@@ -1201,6 +1207,12 @@ void ContextifyContext::CompileFunction(
   if (result->Set(parsing_context, env->cache_key_string(), cache_key)
           .IsNothing())
     return;
+  if (result
+          ->Set(parsing_context,
+                env->source_map_url_string(),
+                fn->GetScriptOrigin().SourceMapUrl())
```

Comment:
Does this work with the debug build? I think this being empty could trigger a DCHECK.

Reply:
I added a test on no magic comment presents. Verified with debug build that no DCHECK are violated.